internal/dkg/store.go

```diff
@@ -199,16 +265,24 @@ func (s *BoltStore) MigrateFromGroupfile(beaconID string, groupFile *key.Group,
 		return errors.New("cannot migrate from groupfile if DKG state exists for beacon")
 	}
 
+	state, err := GroupFileToDBState(beaconID, groupFile, share)
```

Comment:
should this be calling `GroupFileToDBState` or `MigrateFromGroupFile`?

Reply:
`GroupFileToDBState` is a factor-out of logic that was previously inside `MigrateFromGroupFile` and needed to be called from the `fs` method as well.